File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int cnt = 0;
        int res = 0;
        for(int num:nums){
            if(num == 1){
                res = res + 1;
                cnt = Math.max(cnt, res);
            }else{
                res = 0;
            }
        }
        return cnt;
    }
}

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        cnt = 0
        mx = 0
        for ele in nums:
            if ele == 1:
                cnt += 1
                mx = max(mx, cnt)
                print(mx)
            else:
                cnt = 0
        return mx
